File: MyMindNotes/app/src/main/java/com/android/mymindnotes/model/retrofit/JoinApi.java
package com.android.mymindnotes.model.retrofit;

import com.android.mymindnotes.model.UserInfo;

import java.util.Map;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface JoinApi {
    @POST("/api/member/add")
    Call<Map<String, Object>> addUser(@Body UserInfo userinfo);
}

File: MyMindNotes/app/src/main/java/com/android/mymindnotes/data/retrofit/JoinApi.kt
package com.android.mymindnotes.data.retrofit

import retrofit2.http.POST
import com.android.mymindnotes.data.retrofit.model.UserInfo
import retrofit2.http.Body

interface JoinApi {
    @POST("/api/member/add")
    suspend fun addUser(
        @Body userinfo: UserInfo
    ): Map<String, Object>
}

